app/src/main/res/values/strings.xml

```diff
@@ -18,4 +19,9 @@
     <string name="due_date_placeholder">-- / -- / ----</string>
     <string name="save">Save</string>
     <string name="delete">Delete</string>
+    <string name="scan_QR">Scan QR</string>
+    <string name="message_chats">Messages</string>
+    <string name="homeScreen_events">Home</string>
+    <string name="user_profile">Profile</string>
+    <string name="my_hosted_events">My hosting</string>
 </resources>
```

Comment:
nit: it is customary to always have at least one empty line at the end of files.